catalog/spatial/csw/spatial-csw-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/CswRecordMetacardType.java

```diff
@@ -396,16 +396,14 @@ private void addCswBriefRecordAttributes() {
         descriptors.add(new AttributeDescriptorImpl(CSW_TITLE, NON_QUERYABLE /* indexed */, true /* stored */,
                 false /* tokenized */, true /* multivalued */, BasicTypes.STRING_TYPE));
 
-        descriptors
-                .add(new AttributeDescriptorImpl(CSW_ALTERNATIVE, NON_QUERYABLE /* indexed */, true /* stored */,
-                        false /* tokenized */, true /* multivalued */, BasicTypes.STRING_TYPE));
```

Comment:
I know this isn't related to your change, but shouldn't the comment next to `NON_QUERYABLE` be "_not_ indexed"? It looks like this may have been copied & pasted incorrectly in a bunch of places.